diff -Naur acct-6.3.2.old/utmp_rd.c acct-6.3.2/utmp_rd.c
--- acct-6.3.2.old/utmp_rd.c	1997-10-13 20:28:36.000000000 -0400
+++ acct-6.3.2/utmp_rd.c	2012-06-20 11:48:50.000000000 -0400
@@ -118,7 +118,8 @@
       fprintf (out, "%-*.*s|", HOST_LEN, HOST_LEN, rec->ut_host);
 #endif
 
-      fputs (ctime ((time_t *) &(rec->ut_time)), out);
+      time_t uttime = (time_t)rec->ut_time;
+      fputs ( ctime (&uttime), out);
     }
 }
 
